docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,1707 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def _find_discords(T, m, M_T, Σ_T, is_cands):
</pre><p>How do you feel about changing this function name to <code>_refine_candidates</code>. And then update the docstring to "return candidate discords". This way, we reserve the <code>_discords</code> and <code>discords</code> functions for returning actual "discords" or "top K discords" while this function returns "candidate discords" for further "top K" processing.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
I see. I agree with you on changing the name `_find_discords`. I was thinking of something like "_find_candidate_discords"... but this is too long! 

I will apply the proposed changes in the next set of commits.